requirements/testing.txt

```diff
@@ -6,3 +6,6 @@ coverage
 mock==1.0.1
 
 redgreenunittest==0.1.1
+
+# Fixture loading
+PyYAML
```

Comment:
We're pinning other depdencies so that we don't get unexpected major version jumps.